Box.V2/Models/BoxPermission.cs

```diff
@@ -17,7 +17,6 @@ public class BoxPermission
         /// <summary>
         /// Whether the item can be previewed or not
         /// </summary>
-        [Obsolete("CanPreview is now deprecated in the API and will always return true.")]
```

Comment:
I've checked it against specification. There is no mark for obsolete for this item.